index.js

```diff
@@ -379,18 +379,27 @@ module.exports = {
         keyField = key.replace(/:key$/, '');
       }
       if (key && record[key]) {
-        return async.series([findForUpdate, convert, beforeUpdate, update, afterUpdate], outcome);
+        return async.series([ findForUpdate, convert, beforeUpdate, update, afterUpdate ], outcome);
       } else {
-        return async.series([convert, before, insert, after], outcome);
+        return async.series([ convert, before, insert, after ], outcome);
       }
       function outcome(err) {
         // Don't flunk the whole job for one bad row, just report it
         if (err) {
           self.apos.utils.error(err);
+          const displayError = err.message
+            ? err.message.includes('Document has neither slug nor title, giving up')
+            : err.includes('update-notfound');
+
           return self.db.update({ _id: job._id }, {
             $inc: {
               errors: 1,
               processed: 1
+            },
+            ...displayError && {
+              $addToSet: {
+                errorMessages: err.message || err
```

Comment:
This is correct, but is it enough information to help them fix it? In the case of "update-notfound" should we add the value that was given for the update key to the error? The "neither slug nor title" one is probably enough information.

Reply:
Here is an example of a "not-found" error:

![image](https://user-images.githubusercontent.com/11479686/118474503-6f9cc380-b70b-11eb-93fe-2a67db98a107.png)

It displays the key that is supposed to be updated but that does not exist yet (the title in my example). It is what is displayed currently in the terminal anyway, so no more or less information that a server admistrator has.